main.py

```diff
@@ -1,7 +1,23 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
+
+genres = ["Western", "Action", "Dramma"]
+actors = [("George", "Klooney"), ("Kianu", "Reaves"),
+          ("Scarlett", "Keegan"), ("Will", "Smith"),
+          ("Jaden", "Smith"), ("Scarlett", "Johansson")]
 
 
 def main() -> QuerySet:
-    pass
+    for item in genres:
+        Genre.objects.create(name=item)
+    for first, last in actors:
```

Comment:
first, last what?
first item, last item? 
Choose correct name for this variable